package.json

```diff
@@ -87,7 +87,7 @@
     "@cucumber/cucumber": "^10.8.0",
     "@edium/fsm": "^2.1.2",
     "@faker-js/faker": "^9.2.0",
-    "@hmcts/ccd-case-ui-toolkit": "7.1.32",
+    "@hmcts/ccd-case-ui-toolkit": "7.1.37-qm-wa",
```

Comment:
Minor comment - 7.1.37 seems a bit high? Seems to have been scaling with query management though so maybe I'm wrong